sql-plugin/src/main/321db/scala/com/nvidia/spark/rapids/shims/GpuWindowInPandasExec.scala

```diff
@@ -0,0 +1,278 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
```

Comment:
nit: for me if we are just copying a file I don't see a reason to update/modify the copyright.  If the file was written by someone other than NVIDIA we would not remove their copyright and add an NVIDIA one.

Reply:
I think our convention has been to update it. I'm not a lawyer and personally don't care as long as we say what the convention is and keep consistent.

Reply:
The good news is that for this PR, I'm adding a 311+-db directory which removes quite a few redundant copies and nicely side-steps this issue.